File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/SignDocuments/ArubaSignModel.ts
import SignRequestType = require("./SignRequestType");
import SignType = require("./SignType");

class ArubaSignModel {
    ServiceName: string;
    DelegatedDomain: string;
    DelegatedPassword: string;
    DelegatedUser: string;
    OTPPassword: string;
    OTPAuthType: string;
    User: string;
    UserPassword: string;
    CertificateId: string;
    RequestType: SignRequestType;
    SignType: SignType;
}

export = ArubaSignModel;

File: SOURCE_BIBLOSDS/VecompSoftware.Sign/ArubaSignService/Models/ArubaSignModel.cs

using VecompSoftware.Sign.Models;

namespace VecompSoftware.Sign.ArubaSignService
{
    public class ArubaSignModel
    {
        public string ServiceName { get; set; }
        public string DelegatedDomain { get; set; }
        public string DelegatedPassword { get; set; }
        public string DelegatedUser { get; set; }
        public string OTPPassword { get; set; }
        public string OTPAuthType { get; set; }
        public string User { get; set; }
        public string UserPassword { get; set; }
        public string CertificateId { get; set; }
        public string TSAUser { get; set; }
        public string TSAPassword { get; set; }
        public SignRequestType RequestType { get; set; }
    }
}
